.github/workflows/gh-aw-stale-issues.md

```
@@ -89,6 +108,16 @@ Find open issues that are very likely already resolved and recommend them for cl
 
 ### Data Gathering
 
+0. **Read the prescanned issue index**
+
+   A prescan step has already fetched open issues (sorted by least recently updated) into `/tmp/gh-aw/agent/open-issues.tsv` with columns: number, title, updated_at, created_at, label_names. Start by reading this file:
+
+   ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)